changelog/install-dmd-dc.dd

```diff
@@ -0,0 +1,27 @@
+`~/dlang/install.sh` now supports `--dmd` and `--dc`
+
+`--dmd` and `--dc` has been added as a new option to the install script.
+These flags work analogous to the existing `--activate` flag and
+print information about the installed compiler binary:
+
+$(UL
+    $(LI `--dmd` returns the path of a specific compiler executable (DMD-like interface))
+    $(LI `--dc` returns the path of a specific compiler executable (custom compiler interface))
```

Comment:
Can this be the same as what `--dmd` gives?  Might be worth noting if so.

Reply:
I added an example.